src/lib/globals.ts

```diff
@@ -64,6 +65,10 @@ declare global {
     var _azleCanisterMethods: CanisterMethods;
     // eslint-disable-next-line no-var
     var _azleCanisterMethodsIndex: number;
+    // eslint-disable-next-line no-var
+    var _azleCanisterMethodIdlTypes: { [key: string]: IDL.FuncClass };
+    // eslint-disable-next-line no-var
+    var _azleSystemCanisterMethodIdlTypes: IDL.FuncClass[];
```

Comment:
Change to `_azleInitAndPostUpgradeIdlTypes`

Reply:
done